db/models/applications.py

```diff
@@ -149,6 +149,21 @@ def get_all():
         application_list = db.session.query(Applications).all()
         return application_list
 
+    @staticmethod
+    def get_count_by_status():
+        statuses = {s.name: 0 for s in Status}
+        status_query = (
+            db.session.query(
+                Applications.status, func.count(Applications.status)
+            )
+            .group_by(Applications.status)
+            .all()
+        )
+        statuses_with_counts = {
+            status[0].name: status[1] for status in status_query
+        }
+        return statuses | statuses_with_counts
```

Comment:
I have not seen this union operator in python before, nice way to pass 0 count statuses 👍